ftw/contacts/resources/scss/summary.scss

```diff
@@ -64,6 +64,18 @@
     background-color: $color-gray-light;
     background-image: embed-resource("../images/company.svg");
   }
+  &:before {
```

Comment:
We don't need these lines since you use the `clear: both;` approach instead of setting the height manually.